### PR TITLE
Completed Task 001

### PR DESCRIPTION
**Pull request by Hy-light**
All task 001 has been completed as per requirements. 

I am open to all input to have a better code.

### CODE REVIEW

File: sellshoes/accounts/admin.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        (None, {'fields': ('email', 'password')}),
+        ('Personal Info', {'fields': ('first_name', 'last_name')}),
+        ('Permissions', {'fields': ('is_staff', 'is_active', 'is_superuser')}),
+    )
+
+    # Fields to be used in the admin user creation form
+    add_fieldsets = (
+        (None, {
+            'classes': ('wide',),
+            'fields': ('email', 'first_name', 'last_name', 'password1', 'password2', 'is_staff', 'is_active', 'is_superuser'),
```
Comment (believeohiozua) [inline sellshoes/accounts/admin.py:28]: For safety reasons we have to remove passwords from the list, both `password1` and `password2` .

Comment (believeohiozua) [inline sellshoes/accounts/admin.py]: For safety reasons we have to remove passwords from the list, both `password1` and `password2` .

File: sellshoes/sellshoes/settings.py [hunk 2]
```diff
@@ -40,6 +40,9 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
+    
+    # Local apps
+    "accounts",
 ]
 
 INSTALLED_APPS += [
```
Comment (believeohiozua) [inline sellshoes/sellshoes/settings.py:45]: let's add the app we created here..

- Everything we need to add to the  `INSTALLED_APPS` we add them here in `INSTALLED_APPS +=[`
- Although it is the same thing but, it help use demacate our custome config from the defaults.

File: sellshoes/sellshoes/settings.py [hunk 3]
```diff
@@ -40,6 +40,9 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
+    
+    # Local apps
+    "accounts",
```
Comment (believeohiozua) [inline sellshoes/sellshoes/settings.py]: ~accounts~

**Review by believeohiozua — CHANGES_REQUESTED** [review]
overall your work is great, it does not look like you just started django
just this small fixes and it's all good.

**Comment by believeohiozua** [comment]
Please do not forget to `fetch` and `pull` on your next `push` so as to prevent `merge conflict`.

File: sellshoes/accounts/admin.py (hunk 1 above)
Comment (Hy-light) [reply sellshoes/accounts/admin.py]: Ok

**Pull request merged**